web-registry/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -42,8 +42,8 @@ const Dashboard = (): JSX.Element => {
   const isCreditClassCreator = useQueryIfCreditClassCreator();
   const isCreditClassAdmin = useQueryIfCreditClassAdmin();
   const isProjectAdmin = useQueryIfProjectAdmin();
-  const projectTabHidden = !(isIssuer || isProjectAdmin);
-  const creditClassTabHidden = !(isCreditClassCreator || isCreditClassAdmin);
+  const projectTabHidden = !isIssuer || !isProjectAdmin;
```

Comment:
i have a preference for the prior style but i do see this is equivalent and maybe others prefer this style.

Reply:
This is more readable for my brain, but maybe it's just me. At first I thought this was the issue, but it's not. I tend to favor readability in the end.

Reply:
i read the former as “is this not an admin or a creator”. whereas the latter reads “is this not admin or not a creator”. the former reads better in my brain 😆